src/virtualenv/create/via_global_ref/builtin/cpython/mac_os.py

```diff
@@ -70,7 +70,9 @@ def image_ref(cls, interpreter):
 class CPython2macOsFramework(CPythonmacOsFramework, CPython2PosixBase):
     @classmethod
     def can_create(cls, interpreter):
-        return not IS_MAC_ARM64 and super(CPython2macOsFramework, cls).can_describe(interpreter)
+        if not IS_MAC_ARM64 and super(CPython2macOsFramework, cls).can_describe(interpreter):
+            return super(CPython2macOsFramework, cls).can_create(interpreter)
+        return None
```

Comment:
Should be False, not?

Reply:
I didn't precisely pin down the return type of can_create(). I do know it's not strictly `bool`. I chose `None` because it's falsey and it is one value that `ViaGlobalRefVirtualenvBuiltin.can_create()` can return.

I've now tried it with `return False`, and the unit tests still fail in the same way as described above (i.e. this bug is resolved, revealing others). So I'm happy with either.

Reply:
Yeah, let's make it False please :blush:

Reply:
Done